phase1/vsphere/README.md

```diff
@@ -223,22 +223,25 @@ For Kubernetes release 1.6 and above use `docker.io/cnastorage/k8s-ignition:v2`.
 *
 * Phase 2: Node Bootstrapping
 *
-installer container (phase2.installer_container) [docker.io/cnastorage/k8s-ignition:v2] (NEW) docker.io/cnastorage/k8s-ignition:v2
+```
+* Set the release of Kubernetes to be used. The release should be the exact string used to tag a release.
+```
+kubernetes version (phase2.kubernetes_version) [v1.6.5] (NEW)
 ```
 
-* Registry to be used by Kubernetes
+* Set bootstrap provider to ignition
 ```
-docker registry (phase2.docker_registry) [gcr.io/google-containers] (NEW)
+bootstrap provider (phase2.provider) [ignition] (NEW) ignition
 ```
 
-* Set the release of Kubernetes to be used. The release should be the exact string used to tag a release.
+* Set the installer container
 ```
-kubernetes version (phase2.kubernetes_version) [v1.6.5] (NEW) v1.6.5
+installer container (phase2.installer_container) [docker.io/cnastorage/k8s-ignition:v2] (NEW) docker.io/cnastorage/k8s-ignition:v2
 ```
 
-* Set bootstrap provider to ignition
+* Registry to be used by Kubernetes
 ```
-bootstrap provider (phase2.provider) [ignition] (NEW) ignition
+docker registry (phase2.docker_registry) [gcr.io/google-containers] (NEW)
```

Comment:
Currently, this is before installer container. Can you please change this?

Reply:
I copy this order this directly from the install script, is that wrong?

Reply:
ditto